imagersite/imager_images/models.py

```diff
@@ -42,10 +47,8 @@ def __str__(self):
         return "{}: Album belonging to {}".format(self.title, self.user)
 
     def __repr__(self):
```

Comment:
SQLAlchemy wants you to use `__repr__`, but in Django the preference is for using `__str__`